### PR TITLE
Adds nextjs-typeorm sample app

### PR DESCRIPTION
**Pull request by DavidSeptimus-Klotho**

### CODE REVIEW

File: ts-nextjs-typeorm/.gitignore [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (yuval-klotho) [inline ts-nextjs-typeorm/.gitignore:1]: ❔  rm this file? or move it to the repo root?

File: ts-nextjs-typeorm/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (yuval-klotho) [inline ts-nextjs-typeorm/README.md:1]: Maybe also point to our tutorial?

File: ts-nextjs-typeorm/external-content.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
+
+import * as fs from "fs/promises"
+import * as express from "express";
+
+export const router = express.Router();
+
+
+router.use(express.static("external-content"))
+// router.use('/external-content', express.static("external-content"))
```
Comment (yuval-klotho) [inline ts-nextjs-typeorm/external-content.ts:10]: Is this something that the tutorial will have us uncomment? If not, we should rm it. If yes, I think a comment would be great (to point them to the tutorial and give a brief explanation of what this commented-out bit is for).

Ditto on line 20.

File: ts-nextjs-typeorm/external-content.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+export async function nextExternalContentMiddleware(req, res, next) {
+  let file = req.query.url;
+  if (file.startsWith('/external-content')) {
+    file = file.substring(1)
+    const content = await fs.readFile(`${file}`)
+    res.contentType(file)
+    return res.send(content)
+  }
+
+  next()
```
Comment (yuval-klotho) [inline ts-nextjs-typeorm/external-content.ts:36]: does this return automatically? Or if not, does it need to be `await`ed?

**Review by yuval-klotho — COMMENTED** [review]
lgtm, just a few questions. I gave most of the files the briefest of scroll-throughs, under the assumption that they're generated/provided and don't actually need a review.

File: ts-nextjs-typeorm/external-content.ts (hunk 4 above)
Comment (DavidSeptimus-Klotho) [reply ts-nextjs-typeorm/external-content.ts:36]: Ah, I forgot to remove this file.

File: ts-nextjs-typeorm/README.md (hunk 2 above)
Comment (DavidSeptimus-Klotho) [reply ts-nextjs-typeorm/README.md:1]: Yeah, this just the readme from the template. I haven't spent time on that for this sample.

**Review by yuval-klotho — APPROVED** [review]

**Pull request merged**